superbuild/External-Boost.cmake

```diff
@@ -29,13 +29,25 @@ else()
   set(Boost_address_model 32)
 endif()
 
+if (MSVC_VERSION EQUAL 1500) #VS2008
+set(boost_toolset "msvc-9.0")
+elseif(MSVC_VERSION EQUAL 1600) #VS2010
+set(boost_toolset "msvc-10.0")
+elseif(MSVC_VERSION EQUAL 1700) #VS2012
+set(boost_toolset "msvc-11.0")
+elseif(MSVC_VERSION EQUAL 1800) #VS2013
+set(boost_toolset "msvc-12.0")
+elseif(MSVC_VERSION EQUAL 1900) #VS2015
+set(boost_toolset "msvc-14.0")
+endif(MSVC_VERSION EQUAL 1500)
+
 ExternalProject_Add(Boost
   BUILD_IN_SOURCE 1
   URL ${Boost_url}
   URL_MD5 ${Boost_md5}
   UPDATE_COMMAND ""
   CONFIGURE_COMMAND ${Boost_Bootstrap_Command} --prefix=${INSTALL_DEPENDENCIES_DIR}/lib
-  BUILD_COMMAND ${Boost_b2_Command} install -j8   --prefix=${INSTALL_DEPENDENCIES_DIR} --with-thread --with-filesystem --with-system --with-date_time --with-program_options address-model=${Boost_address_model}
+  BUILD_COMMAND ${Boost_b2_Command} install -j8   --prefix=${INSTALL_DEPENDENCIES_DIR} --with-thread --with-filesystem --with-system --with-date_time --with-program_options address-model=${Boost_address_model} --toolset=${boost_toolset}
```

Comment:
When someone does not use MSVC, the the BUILD_COMMAND will end with '--toolset=' ??
wouldn't it be better to store "--toolset=" into one variable only set for provided MSVC_VERSION ?